src/ar_tx.erl

```diff
@@ -78,7 +78,12 @@ sign(TX, PrivKey, PubKey) ->
 %% submitting an arbitrary length signature field and having it processed.
 -ifdef(DEBUG).
 verify(#tx { signature = <<>> }, _, _) -> true;
-verify(TX, Diff, WalletList) ->
+verify(TX, Diff, WalletList) -> do_verify(TX, Diff, WalletList).
+-else.
+verify(TX, Diff, WalletList) -> do_verify(TX, Diff, WalletList).
+-endif.
+
+do_verify(TX, Diff, WalletList) ->
 	% ar:report(
```

Comment:
May be let's remove this comment as well?

Reply:
✅